test/mocks/server/mocks.h

```diff
@@ -516,11 +516,11 @@ class MockListenerFactoryContext : public MockFactoryContext, public ListenerFac
   MockListenerFactoryContext();
   ~MockListenerFactoryContext() override;
 
-  void addListenSocketOption(const Network::Socket::OptionConstSharedPtr& option) override {
+  void addListenSocketOption(const Network::Socket::OptionConstSharedPtr& option) {
```

Comment:
All of this can be removed from the mocks I think.

Reply:
removed and no complain from bazel